client/youtube-genre.html

```diff
@@ -4,19 +4,42 @@
     <meta charset="UTF-8">
     <title>Youtube Genre</title>
     <link rel="stylesheet" href="css/base.css">
+    <link rel="stylesheet" href="css/scroll.css">
   </head>
   <body>
-    <div id="content">
-      <h1>What music do you listen to on youtube?</h1>
-      <div id="numVideoForm">
-        <p>
-          Choose number of liked videos to retrieve:
-          <input type="number" name="numVideos" id="numVideos" value="10" placeholder="Number of Liked Videos"/>
-          <input type="submit" onclick="displayMusicGenre()"/>
-        </p>
-      </div>
+    <div class="title hero" id="banner"> 
+      <h1>What music do you listen to on YouTube? </h1>
+      <p>
+        Choose number of liked videos to retrieve:
+        <input type="number" name="numVideos" id="numVideos" value="10" placeholder="Number of Liked Videos"/>
+        <input type="submit" onclick="displayMusicGenre()" class="btn"/>
+      </p>
+      <div id="genres"></div>  
     </div>
-    <ul id="genres"></ul>
+    <main>
+      <div id='graphic'>
+        <div id='sections'>
+          <section class="step">      
+          </section>
+          <section class="step">
+            Section 2
+          </section>
+          <section class="step">
+            Section 3
+          </section>
+          <section class="step">
+            Section 4
+          </section>
+          <section class="step">
+            Section 5
+          </section>
+        </div>
+        <div id='vis'>
+          add visualizations here
+        </div>
+      </div>
+    </main>
   </body>
+  <script src="https://d3js.org/d3.v4.min.js"></script>
```

Comment:
Why are you using v4 instead of v5?

Reply:
just because the d3 examples I'm looking at are using v4 😟 

I'll change to v5 but worried about deprecated functions